integration-tests/deployment/ccip/deploy.go

```diff
@@ -153,46 +176,27 @@ func DeployCCIPContracts(e deployment.Environment, c DeployCCIPContractConfig) (
 			e.Logger.Errorw("Failed to load chain state", "err", err)
 			return ab, err
 		}
-		// Enable ramps on price registry/nonce manager
-		tx, err := chainState.PriceRegistry.ApplyAuthorizedCallerUpdates(chain.DeployerKey, fee_quoter.AuthorizedCallersAuthorizedCallerArgs{
-			// TODO: We enable the deployer initially to set prices
-			AddedCallers: []common.Address{chainState.EvmOffRampV160.Address(), chain.DeployerKey.From},
-		})
-		if _, err := deployment.ConfirmIfNoError(chain, tx, err); err != nil {
-			e.Logger.Errorw("Failed to confirm price registry authorized caller update", "err", err)
-			return ab, err
-		}
-
-		tx, err = chainState.NonceManager.ApplyAuthorizedCallerUpdates(chain.DeployerKey, nonce_manager.AuthorizedCallersAuthorizedCallerArgs{
-			AddedCallers: []common.Address{chainState.EvmOffRampV160.Address(), chainState.EvmOnRampV160.Address()},
-		})
-		if _, err := deployment.ConfirmIfNoError(chain, tx, err); err != nil {
-			e.Logger.Errorw("Failed to update nonce manager with ramps", "err", err)
-			return ab, err
-		}
 
+		// TODO: Do we want to extract this?
 		// Add chain config for each chain.
-		_, err = AddChainConfig(e.Logger,
+		_, err = AddChainConfig(
+			e.Logger,
 			e.Chains[c.HomeChainSel],
 			c.Chains[c.HomeChainSel].CCIPConfig,
 			chain.Selector,
-			nodes.PeerIDs(chain.Selector),
-			uint8(len(nodes)/3))
+			nodes.PeerIDs(chain.Selector))
 		if err != nil {
 			return ab, err
 		}
 
-		// For each chain, we create a DON on the home chain.
-		if err := AddDON(e.Logger,
-			cr,
+		// For each chain, we create a DON on the home chain (2 OCR instances)
+		if err := AddDON(
+			e.Logger,
 			c.Chains[c.HomeChainSel].CapabilityRegistry,
 			c.Chains[c.HomeChainSel].CCIPConfig,
-			chainState.EvmOffRampV160,
+			chainState.OffRamp,
 			chain,
 			e.Chains[c.HomeChainSel],
-			uint8(len(nodes)/3),
-			nodes.BootstrapPeerIDs(chain.Selector)[0],
-			nodes.PeerIDs(chain.Selector),
 			nodes,
```

Comment:
Can you change this send plugin nodes only? with real JD if it includes bootstrap it will fail for having blank OCR2 keys and transmitter account